packages/paste-core/components/skeleton-loader/src/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useSpring, animated} from '@twilio-paste/animation-library';
+import {css, styled} from '@twilio-paste/styling-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {LayoutProps, MarginProps, BorderRadiusProps} from '@twilio-paste/style-props';
+
+const AnimatedSkeleton = animated(Box);
+
+const StyledAnimatedSkeleton = styled(AnimatedSkeleton)(() =>
+  css({
+    background: `linear-gradient(90deg, transparent, rgba(255, 255, 255, 0.3) 40%, rgba(255, 255, 255, 0.3) 60%, transparent)`,
+  })
```

Comment:
I wish we had a better solution for these types of situations. They aren't using system values, and compound css values are really difficult to systemize (but potentially not impossible).

Something to think about for the future.

Reply:
Gradient tokens 👓

Reply:
I like the idea of a gradient function. Maybe we could add that as library or shared utility function in the future.